Project.toml

```diff
@@ -19,3 +19,15 @@ Knet = "1.4.9"
 CUDA = "3.11.0"
 MLDatasets = "0.5.14"
 NLPModels = "0.18, 0.19"
+
+[extras]
+CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
+IterTools = "c8e1da08-722c-5040-9ed9-7db0dc04731e"
+Knet = "1902f260-5fb4-5aff-8c31-6271790ab950"
+MLDatasets = "eb30cadb-4394-5ae3-aed4-317e484a6458"
+NLPModels = "a4795742-8479-5a88-8948-cc11e1c8c1a6"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
Don't repeat dependencies. But I don't thin CUDA, etc, are dependencies, so they should be removed from above.